### ISSUE TITLE
False negative in some control flows

### ISSUE THREAD

**Issue by sobolevn** [gforcada/flake8-builtins#17]
Given this code sample:

```python
try:
     1 / 0
except ZeroDivisionError as sum:
    print(sum)
```

This should rise an error. But it does not.
That's a bug, I guess. 

I will create a PR with the test after #15 will be merged.

**Comment by gforcada** [comment]
Yep, I was looking right now to which other control flow statements allow to define variables.

Meanwhile I pushed a 1.1 version with the previous fix. Thanks for reporting them!

**Comment by gforcada** [comment]
To get a complete *(hopefully)* list of other flow statements:
- except as
- with

Anything else? :thinking:

**Comment by sobolevn** [comment]
`import as`, `class`, `def`, `async def`, `async with`, `async for`
And also function and method parameters.

That's all, I guess.

**Comment by sobolevn** [comment]
And I have forgot `class` attributes.

**Comment by gforcada** [comment]
@sobolevn flake8-builtins 1.2.1 is out with tons of more proves, let me know if it works as expected.

**Comment by sobolevn** [comment]
Great! Thanks!
